blog/2023-07-18-wing-cloud-launch.md

```diff
@@ -0,0 +1,281 @@
+the flat surfaces, etc. My gate is built out of reds and blues and greens and a bunch of cables. But
+I can't see a gate. I just see the building blocks from which it's constructed. How would I know
+which blue piece to move and where in order to open the gate?
+
+This is the experience we have today when we operate applications in the cloud. We get an alarm that
+there are some errors from some Lambda function. How do we know which part of system is affected?
+Which team owns this part?
+
+The solution, today, is dashboards. I have to manually craft these views to represent the mental
+model of my system. I take the red and green and blue pieces and arrange them together under the
+title "Gate", and now if there's an error, I can tell where it belongs.
+
+It goes back to the *space and time* analogy. The operational experience of Wing Console is based on
+Wing's *spacial view* of the system. Every resource or event can be *traced back* to the logical
+unit it belongs to. This is basically the purpose of stack traces in traditional languages - an
+exception is thrown and the stack trace helps me trace back from low-level to high-level so I can
+diagnose the problem.
+
+### Fully customizable below the abstraction
+
+We are not naive to think that we can create an abstraction that will cover all use cases and
+possible setups. In fact, Wing is designed from the ground up to *celebrate cloud choice and
+diversity* and includes multiple levels of extensibility and customizability.
+
+One dimension of extensibility is support for any resource from the IAC ecosystem of the target as
+first-class citizens. For example, if you compile you Wing code to Terraform (`tf-aws`, `tf-azure`,
+`tf-gcp`), then you can use any resource in the [Terraform Registry](registry.terraform.io) in your
+Wing code. If you compile to AWS CDK, you can use any [AWS CDK
+construct](https://docs.aws.amazon.com/cdk/api/v2/) in your code (L1s, L2s and any construct library
+from the [Construct Hub](https://constructs.dev/)). Naturally, if you use any of these resources,
+they will only work when deploying to the cloud, but there is built-in support in Wing for providing
+a local implementation for them.
+
+Another dimension is what we call [Wing Compiler
+Plugins](https://www.winglang.io/docs/tools/compiler-plugins). Those are hooks implemented through
+JavaScript functions that can customize the output of the Wing compiler. These plugins are extremely
+powerful and work **below the abstraction line**. This means that they can utilize capabilities that
+are provider-specific, **enforce policies**, **apply compliance rules**, etc. We believe these
+plugins can be used by platform and centralized cloud teams to enforce and apply common standards
+within a company in a streamlined ways for developers.
+
+### We are just getting started
+
+We believe in the cloud. We believe in it's economic sense, it's diversity and choice. We believe
```

Comment:
```suggestion
We believe in the cloud. We believe in its economic sense, its diversity and choice. We believe
```

Reply:
diversity and choice sort of sound like the same thing on my first read, maybe use a different third thing? not sure

Reply:
rephrased